### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sunnngirl**

### CODE REVIEW

File: README.md
Comment (tianyizheng02) [inline README.md:89]: This link looks suspicious to me. Tokio's official site is https://tokio.rs, not https://tokio.uk.ms, and I can't confirm whether the latter is an official, alternative domain. Furthermore, there's no equivalent careers site on the official .rs site. In fact, the .uk.ms site is a perfect clone of the official .rs site _except_ that the Blog tab is replaced with a Careers tab. For security reasons, please change the link to a careers page on the official Tokio site instead.

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

**Review by owini — COMMENTED** [review]
I wouldn't merge.

File: README.md
Comment (ewang2002) [reply README.md:89]: Looks like you already took the initiative to ask on Tokio's official GitHub page. 

The fact that this page looks identical to the official page makes me skeptical that this is an official Tokio website. Also, I feel like if this was real, they would put more effort into ensuring that their application system works — right now, you could put nothing in and press submit, and it would blindly accept it. 

I wouldn't merge.

Comment (itsk3vin) [reply README.md:89]: TIL why I should wait until an application is merged to apply

**Comment by tianyizheng02** [comment]
Opened a discussion on Tokio's official GitHub about it, currently awaiting a reply. We can merge/close this PR once we get confirmation

File: README.md
Comment (tianyizheng02) [reply README.md:89]: @kevinroosey TBF, it's possible that we maintainers accidentally let some suspicious postings through from time to time, so I definitely encourage taking the initiative to verify postings independently (and please open a PR if you find something!)

**Review by jacksonsmitg207 — APPROVED** [review]
Cool

**Review by ewang2002 — COMMENTED** [review]
The link is no longer functional (i.e., https://tokio.uk.ms/careers redirects to https://tokio.rs). As such, I'm closing this PR. 

If you find any other internship offerings, please do not hesitate to create another PR though. Thanks!

**Comment by tianyizheng02** [comment]
> The link is no longer functional (i.e., https://tokio.uk.ms/careers redirects to https://tokio.rs).

@ewang2002 ... did we help to get the site taken down? lmao

**Comment by ewang2002** [comment]
> > The link is no longer functional (i.e., https://tokio.uk.ms/careers redirects to https://tokio.rs).
> 
> @ewang2002 ... did we help to get the site taken down? lmao

No idea, but I'm not questioning it 🙃 Glad it's been resolved 😄

**Comment by Darksonn** [comment]
Thank you for being vigilant and checking with us before merging something like this.

**Pull request closed without merging**
